templates.rst

```diff
@@ -818,6 +818,13 @@ errors. It's useful to run it before deploying your application to production
     # you can also show the deprecated features used in your templates
     $ php bin/console lint:twig --show-deprecations templates/email/
 
+    # you can also excludes directories
```

Comment:
What about using this comment to also explain how to exclude multiple directories? Or add a second example for that?

Reply:
yes indeed lets add 2 to represents the array of dirs